src/components/Card.js

```diff
@@ -15,22 +25,32 @@ export default class Card {
 
   //event listeners for cards
   _setEventListeners() {
-    this._likeButton.addEventListener("click", this._likeCard);
-    this._deleteButton.addEventListener("click", this._removeCard);
+    this._likeButton.addEventListener("click", () => {
+      this.handleLikeClick(this);
+      this._checkedIfLiked();
```

Comment:
Needs correction: you should update the likes state only after the successful api response in then block in index.js. You should have a setter for the likes state:

```js
// call this method from index.js after successful api response in `then` block
setIsLiked(isLiked) { 
   // set instance variable
    this._isLiked = isLiked; 
    this._renderLikes(); 
} 
```